src/overlays/actors/ovl_Boss_Ganon/z_boss_ganon.c

```diff
@@ -4805,17 +4805,20 @@ static void* sLightningTextures[] = {
     gGanondorfLightning12Tex,
 };
 
-static u8 sLightningPrimColors[] = {
-    0,   0,   0,   255, 255, 255, 231, 250, 231, 208, 245, 208, 185, 240, 185, 162, 235, 162, 139, 230,
-    139, 115, 225, 115, 92,  220, 92,  69,  215, 69,  46,  210, 46,  23,  205, 23,  0,   200, 0,
+static u8 sLightningPrimColors[13][3] = {
+    { 0, 0, 0 },       { 255, 255, 255 }, { 231, 250, 231 }, { 208, 245, 208 }, { 185, 240, 185 },
+    { 162, 235, 162 }, { 139, 230, 139 }, { 115, 225, 115 }, { 92, 220, 92 },   { 69, 215, 69 },
+    { 46, 210, 46 },   { 23, 205, 23 },   { 0, 200, 0 },
 };
 
-static u8 sLightningEnvColors[] = {
-    0,   0,   0,   255, 255, 0,   240, 231, 23,  226, 208, 46,  212, 185, 69,  198, 162, 92,
-    184, 139, 115, 170, 115, 139, 156, 92,  162, 142, 69,  185, 128, 46,  208, 114, 23,  231,
-    100, 0,   255, 0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,
+static u8 sLightningEnvColors[13][3] = {
+    { 0, 0, 0 },      { 255, 255, 0 },   { 240, 231, 23 },  { 226, 208, 46 }, { 212, 185, 69 },
+    { 198, 162, 92 }, { 184, 139, 115 }, { 170, 115, 139 }, { 156, 92, 162 }, { 142, 69, 185 },
+    { 128, 46, 208 }, { 114, 23, 231 },  { 100, 0, 255 },
 };
 
+static u8 sUnknown[0xC] = { 0 };
```

Comment:
```suggestion
static s32 sUnknown[3] = { 0 };
```
we prefer s32 for typing padding stuff when possible